content/concepts/nat/autonat.md

```diff
@@ -1,27 +1,53 @@
 ---
 title: "AutoNAT"
-description: AutoNAT lets peers request dial-backs from peers providing the AutoNAT service.
+description: "AutoNAT lets nodes discover if they are behind a NAT."
 weight: 200
 aliases:
     - "/concepts/autonat"
     - "/concepts/nat/autonat"
 ---
 
+## Background
+
+While the [identify protocol][spec_identify] lets peers inform each other about their observed network
+addresses, however, it is possible that some of these addresses are not accessible from outside the network, as the peer may be located in a private network that is behind a [NAT](overview.md) and therefore unreachable.
+
+To prevent this problem of advertising undialable addresses, libp2p has implemented a protocol called AutoNAT, which allows nodes to determine
+whether or not they are behind a NAT.
```

Comment:
This is more about determining your position and then taking specific action based on that, for example switching to DHT server mode, or getting a reservation with a relay.
AutoNAT current can't test _individual_ addresses (that's what my AutoNAT v2 proposal would add).